controllers/tasks.js

```diff
@@ -254,13 +254,13 @@ const getSelfTasks = async (req, res) => {
 const getTask = async (req, res) => {
   try {
     const taskId = req.params.id;
-    const { taskData, dependencyDocReference } = await tasks.fetchTask(taskId);
+    const { taskData, dependencyDocReference, isPendingExtensionRequest } = await tasks.fetchTask(taskId);
```

Comment:
- I think a more suitable term here is `isExtensionRequestPending` as the pending status is a part of the extension request not a by product.